DESCRIPTION

```diff
@@ -1,6 +1,6 @@
 Package: sircovid
 Title: SIR Model for COVID-19
-Version: 0.11.14
+Version: 0.11.15
```

Comment:
please can you add to the NEWS file as well? thx

Reply:
This will need to be .16 now sorry :(